kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -410,20 +411,29 @@ def kompilation_up_to_date() -> bool:
 
         return old_digest == kompilation_digest()
 
+    def kompile_haskell() -> None:
+        _LOGGER.info(f'Kompiling definition: {foundry_main_file}')
+        _kompile(foundry_definition_dir, KompileTarget.HASKELL)
+
+    def kompile_llvm() -> None:
+        _LOGGER.info(f'Kompiling definition to LLVM dy.lib: {foundry_main_file}')
```

Comment:
Is `dy.lib` the file name on all supported platforms?

Reply:
@tothtamas28  According to the haskell backend team, it is `interpreter.so` on linux and `interpreter.dylib`.

Reply:
I see, consider this then:

```
_LOGGER.info(f'Kompiling definition to LLVM dynamic library: {foundry_main_file}')
```

Reply:
Updated the logging message.